provisioning_templates/snippet/ntp.erb

```diff
@@ -0,0 +1,10 @@
+<%#
+kind: snippet
+name: ntp
+model: ProvisioningTemplate
+snippet: true
+%>
+echo "updating system time"
+yum -y install ntpdate
+/usr/sbin/ntpdate -sub <%= host_param('ntp-server') || '0.fedora.pool.ntp.org' %>
+/usr/sbin/hwclock --systohc
```

Comment:
One more thing once you on that. In the default kickstart we have this:

```
  use_ntp = host_param_true?('use-ntp') || (is_fedora && os_major < 16) || (rhel_compatible && os_major <= 7)

<% if use_ntp -%>
ntp
-chrony
<% else -%>
chrony
-ntp
<% end -%>
```

Esssentially, on older systems we do install ntp but on modern ones we do default to chrony, unless user opts-into the traditional ntp. Can you respect the very same in the snippet? Copy the check over and install either ntp or chrony. I'd also enable start the service as well: `systemctl enable --now service`.

Reply:
Beware that `ntpdate` will not work if chrony is installed, use `chronyc -a makestep` instead.

Reply:
Also, I don't like the fedora default pool, I'd probably just provide it an ntp-server if there is any, otherwise i'd keep the default server list that has been installed with the package.